teams_accounts.go

```diff
@@ -62,6 +63,10 @@ type TeamsTLSDecrypt struct {
 	Enabled bool `json:"enabled"`
 }
 
+type TeamsProtocolDetection struct {
+	Enabled bool `json:"enabled"`
```

Comment:
i do recommend all booleans as pointers but up to up folks - https://github.com/cloudflare/cloudflare-go/blob/master/docs/conventions.md#booleans